src/Components/SavedSearchAlert/SavedSearchAlertModal.tsx

```diff
@@ -65,6 +135,10 @@ export const SavedSearchAlertModal: FC<SavedSearchAlertFormProps> = ({
   const isFallbackToGeneratedAlertNamesEnabled = useFeatureFlag(
     "onyx_force-fallback-to-generated-alert-names"
   )
+  const isHearFromArtsyAdvisorEnabled = useFeatureFlag(
+    "onyx_advisory-opportunity-in-saved-search"
+  )
+  const { phone, regionCode } = useUserPhoneNumber()
```

Comment:
There were several approaches that are possible here.

One is that this modal, or possibly just the advisory opportunity / phone number part, should become a fragment container. However, that requires querying fields on `me`, and including that fragment everywhere this button is rendered.

Another approach is to take this modal or a sub-component of the advisory opportunity / phone number form, and make it a query renderer (so it can fetch the user phone information) on its own. That _might_ be more conventional than the approach taken in this PR - I'm not strongly opinionated, although the goal is to make using this shared modal (kicked off by a 'Create Alert' button) easy anywhere. Perhaps that's possible with a query renderer? Can try that or as an iteration.

The approach in this PR to try to minimize updating any callers was allowing this component to fetch the user's phone number (and country code) and inject it into the modal.